data/net/minecraft/world/inventory/MerchantMenu.mapping

```diff
@@ -1,22 +1,39 @@
 CLASS net/minecraft/world/inventory/MerchantMenu
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+	METHOD <init> (ILnet/minecraft/world/entity/player/Inventory;Lnet/minecraft/world/item/trading/Merchant;)V
+		ARG 1 containerId
+		ARG 2 playerInventory
+		ARG 3 merchant
 	METHOD canTakeItemForPickAll (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/inventory/Slot;)Z
 		COMMENT Called to determine if the current slot is valid for the stack merging (double-click) code. The stack passed in is null for the initial slot that was double-clicked.
 		ARG 1 stack
 		ARG 2 slot
+	METHOD isSameItem (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/item/ItemStack;)Z
+		ARG 1 stack1
+		ARG 2 stack2
+	METHOD moveFromInventoryToPaymentSlot (ILnet/minecraft/world/item/ItemStack;)V
+		ARG 1 selectedMerchantRecipe
+		ARG 2 inventorySlot
```

Comment:
These two seem to be wrong, as the code more indicates moving the type of item indicated by the stack from the player's inventory to the slot in the merchant/trader inventory represented by the given slot index. Perhaps `paymentSlotIndex` and `paymentStack`?